projects/Mallard/src/helpers/settings/defaults.ts

```diff
@@ -7,7 +7,7 @@ This is a bit of a mess
 export const backends = [
     {
         title: 'PROD published',
-        value: 'https://editions-store.s3-eu-west-1.amazonaws.com/',
+        value: 'https://editions.guardianapis.com/',
```

Comment:
👍🏻

Reply:
Going to do just CODE to start with then we can ship the change for PROD too once we've confirmed that it's not going to adversly affect the app.